rwkv/rwkv_tokenizer_test.py

```diff
@@ -0,0 +1,722 @@
+from rwkv_tokenizer import get_tokenizer
+
+tokenizer, tokenizer_encode = get_tokenizer("world")
+
+print('Unit test...')
+
+# Test string taken from https://github.com/BlinkDL/ChatRWKV/blob/main/tokenizer/rwkv_tokenizer.py
+test_string = '''
```

Comment:
Oof, I imagined the test to be a simple sanity check on a short string [like this](https://gist.github.com/saharNooby/bb54519a7d3735afb6949825608c00f0#file-simple_bpe_tokenizer-py-L401).

But I've delayed you for long enough. If you prefer, you can replace the test string with that from the link above, which will make the test way shorter. If not, I'm okay with merging it as is and changing it later myself.